src/test/java/guitests/BoardAutoCreatorTest.java

```diff
@@ -72,4 +72,43 @@ public void milestoneBoardAutoCreationTest() {
         assertEquals(panelInfos.get(4).getPanelName(), "Next Next Next Milestone");
     }
 
+
+    @Test
+    public void workAllocationBoardAutoCreationTest() {
+        assertEquals(panelControl.getNumberOfSavedBoards(), 0);
+
+        click("Boards");
+        push(KeyCode.DOWN); // Save
+        push(KeyCode.DOWN); // Save As
+        push(KeyCode.DOWN); // Open
+        push(KeyCode.DOWN); // Delete
+        push(KeyCode.DOWN); // Auto-create
+        push(KeyCode.RIGHT); // Open Auto-create, Milestones
+        push(KeyCode.DOWN); // Work Allocation
+        push(KeyCode.ENTER);
```

Comment:
This is going to break when the menu changes in future. May be write a function that can find an item and click it irrespective of menu content? e.g. `clickMenu(String... items)` and call it `clickMenu("Boards","Auto-create","Work distribution")` Note the use of variable length argument lists.